src/view/emojiCompletionProvider.ts

```diff
@@ -0,0 +1,65 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { ensureEmojis, getEmojis } from '../common/emoji';
+import { Schemes } from '../common/uri';
+
+export class EmojiCompletionProvider implements vscode.CompletionItemProvider {
+	private static readonly ID: string = 'EmojiCompletionProvider';
```

Comment:
Looks like this ID isn't used anywhere. In other classes, this is used when calling the logger, so that we can see where the log is coming from.